shared/chat/conversation/messages/attachment/image/image-render.native.js

```diff
@@ -1,11 +1,69 @@
 // @flow
 import * as React from 'react'
+import {WebView} from 'react-native'
+import WKWebView from 'react-native-wkwebview-reborn'
 import {NativeImage, NativeDimensions} from '../../../../../common-adapters/native-wrappers.native'
```

Comment:
WebView is exported as NativeWebView in native-wrappers.native. outside of common-adapters we shouldn't really import react-native directly